src/main/app/src/app/csv/csv.module.ts

```diff
@@ -0,0 +1,21 @@
+/*
+ * SPDX-FileCopyrightText: 2022 Atos
+ * SPDX-License-Identifier: EUPL-1.2+
+ */
+
+import {NgModule} from '@angular/core';
+import {SharedModule} from '../shared/shared.module';
+import {CsvExportButtonComponent} from './csv-export-button/csv-export-button.component';
+
+@NgModule({
+    declarations: [
+        CsvExportButtonComponent
+    ],
+    exports: [
+        CsvExportButtonComponent
+    ],
+    imports: [
+        SharedModule
+    ]
+})
+export class CsvModule {}
```

Comment:
Een complete module voor een button lijkt me een beetje overbodig. Ik kan zo ook niet bedenken wat er nog meer in een 'csv' module zou moeten staan. Het export-button.component kan gewoon in de SharedModule ondergebracht worden.

Reply:
Verwijderd en verplaatst naar Shared module in nieuwe commit